rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_fluid_loop_purchased_CHW.py

```diff
@@ -0,0 +1,47 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.utils.jsonpath_utils import find_exactly_one_with_field_value
+
+EXTERNAL_FLUID_SOURCE = schema_enums["ExternalFluidSourceOptions"]
+
+
+def is_hvac_sys_fluid_loop_purchased_chw(rmi_b, hvac_b_id):
+    """Returns TRUE if the fluid loop associated with the cooling system associated with the HVAC system is attached to an external purchased chilled water loop. Returns FALSE if this is not the case.
+    Parameters
+    ----------
+    rmi_b : json
+        RMD at RuleSetModelInstance level
+    hvac_b_id : str
+        The HVAC system ID.
+    Returns
+    -------
+    bool
+        True: the fluid loop associated with the cooling system associated with the HVAC system is attached to an external purchased cooling loop
+        False: otherwise
+    """
+    is_hvac_sys_fluid_loop_purchased_chw_flag = False
```

Comment:
Line 21 has no affect because _ is_hvac_sys_fluid_loop_purchased_chw_flag_ is set on line 42 regardless.
So Line 21 should be deleted.